settings/R/MultiSettings.R

```diff
@@ -0,0 +1,195 @@
+##' Create a PEcAn MultiSettings object
+##'
+##' 
+##' @title Create a PEcAn MultiSettings object
+##' @param 
+##' @return
+##' @export
+##' @author Ryan Kelly
+MultiSettings <- function(...) {
+  result <- list(...)
+
+  if(length(result) == 1) {
+    if(is.MultiSettings(result[[1]])) {
+      return(result[[1]])
+    } else if(is.list(result[[1]]) && all(sapply(result[[1]], is.Settings))) {
+      result <- result[[1]]
+    }
+  }
+  
+  if(!all(sapply(result, is.Settings))) {
+    stop("MultiSettings can only be made from Setting, MultiSettings, or a list of Settings")
+  }
+
+  if(length(result) > 0)
+    names(result) <- paste("settings", seq_along(result), sep=".")
```

Comment:
Good catch. Nope, there's nothing special about the MultiSettings' names, and this line was just meant to name them for convenience assuming (incorrectly) that you'd supplied a list of settings (so result would have no names). But you're right I meant for this constructor to be able to simply convert a list of Settings into a MultiSettings, in which case it would be appropriate to leave any names in place. I'll amend the PR in a minute with a fix that only adds names if there are none already.